app/Config/Format.php

```diff
@@ -1,54 +1,59 @@
-<?php namespace Config;
```

Comment:
We're currently all over the place on these. Let's see if @lonnieezell wants to make a call on the styling of namespaces before we change them.

Reply:
I agree with this. Currently it's a mix, and we need a definitive style guide for this one.

Reply:
My personal preference has always been on line 1, but it seems my IDE and lots of other programmers prefer it with a blank space after the opening php tag. I'm fine with either of those options. Not of fan of having the CI docblock above it, though. I think namespace should be the first item in the file.

So do either of you have preferences on first line vs third line?

Reply:
I've been doing it on first line but I think I picked that up from you ;) I don't use an IDE so I'm not sure how much work it is to fight against one. Are there any standards? Does our CS have options that might indicate a "default"?

Reply:
I am following [PSR-12](https://www.php-fig.org/psr/psr-12/).

> When the opening <?php tag is on the first line of the file, it MUST be on its own line with no other statements unless it > is a file containing markup outside of PHP opening and closing tags.

Reply:
I'm good with following the PSR on this. I had forgotten 12 got released, actually, so thanks for the reminder.

Reply:
Oh that's great! Yes we should definitely move towards that then. The consensus is third line for namespace then?

Reply:
Yes. Third line for namespace if no file-level doc block. Namespace after doc block if present. This is following PSR12.

I'll take note on this on my upcoming PR on php-cs-fixer fixes.

Reply:
Bummer - namespace after file docblock is in PSR. Oh  well. Can't have it all :)